Essentials/src/com/earth2me/essentials/Kit.java

```diff
@@ -153,8 +153,8 @@ public void expandItems(final User user, final List<String> items) throws Except
             boolean spew = false;
             final boolean allowUnsafe = ess.getSettings().allowUnsafeEnchantments();
             for (String kitItem : output.getLines()) {
-                if (kitItem.startsWith(ess.getSettings().getCurrencySymbol())) {
-                    BigDecimal value = new BigDecimal(kitItem.substring(ess.getSettings().getCurrencySymbol().length()).trim());
+                if (!ess.getSettings().getCurrencySuffix() ? kitItem.startsWith(ess.getSettings().getCurrencySymbol()) : kitItem.endsWith(ess.getSettings().getCurrencySymbol())) {
+                    BigDecimal value = ess.getSettings().getCurrencySuffix() ? new BigDecimal(kitItem.substring(0, ess.getSettings().getCurrencySymbol().length()).trim()) : new BigDecimal(kitItem.substring(ess.getSettings().getCurrencySymbol().length()).trim());
```

Comment:
![image](https://user-images.githubusercontent.com/3101865/77997890-0aa6c400-72e5-11ea-911b-9aa26e862ce9.png)

This _feels_ messy. I'm trying to think about the right way to get out of this (configuration depending on configuration, liable to break when the one is changed) without torching existing configurations.

Reply:
What are you talking about? This would not break, it's just the ternary operator which is a standard part of java, nothing IMO needs changing here as it works just fine.

Reply:
If you're talking about decimal parsing, that was already part of the code before this and is out of the scope of this PR

Reply:
I'll agree with Joe in that this could at least be split into several lines for readability. Right now you have what is effectively a nested conditional, with the ternary statement inside of an if statement. A little here can go a long way.

As for the configuration dependencies, I don't know how much can be done about it currently. I already approved this PR under the assumption that what you've done is about as good as it will get with how Essentials is currently designed.

Reply:
Its just that the logic for this if got a bit more complicated - maybe its just Github's diff colors, but this got hard to read for me.

```suggestion
                boolean currencyIsSuffix = ess.getSettings().getCurrencySuffix();
                String currencySymbol = ess.getSettings().getCurrencySymbol();
                boolean isCurrency = currencyIsSuffix ? kitItem.endsWith(currencySymbol) : kitItem.startsWith(currencySymbol);
                if (isCurrency) {
                    String valueString = currencyIsSuffix ? kitItem.substring(0, currencySymbol.length()) : kitItem.substring(currencySymbol.length());
                    BigDecimal value = new BigDecimal(valueString.trim());
```

I personally think this reads better since we aren't craming so much into one line.